Documentation/selinux.md

```diff
@@ -1,10 +1,22 @@
 # rkt and SELinux
 
 rkt supports running containers using SELinux [SVirt](http://selinuxproject.org/page/SVirt).
-At start-up, rkt will attempt to read `/etc/selinux/(policy)/contexts/lxc_contexts`.
-If this file doesn't exist, no SELinux transitions will be performed.
-If it does, rkt will generate a per-instance context.
-All mounts for the instance will be created using the file context defined in `lxc_contexts`, and the instance processes will be run in a context derived from the process context defined in `lxc_contexts`.
+The SELinux context for each rkt pod can be set in one of two ways:
 
-Processes started in these contexts will be unable to interact with processes or files in any other instance's context, even though they are running as the same user.
+### Reading SElinux context from the host lxc_contexts file
+
+When invoking `rkt run` or `rkt run-prepared`, rkt will attempt to read `/etc/selinux/(policy)/contexts/lxc_contexts`.
+If this file does not exist, SELinux transitions will be performed.
+Otherwise, rkt will generate a per-pod context.
+All mounts for the pod will be created using the file context defined in `lxc_contexts`, and the processes inside the pod will be run in a context derived from the process context defined in `lxc_contexts`.
+
+Processes started in these contexts will be unable to interact with processes or files in any other pod's context, even though they are running as the same user.
```

Comment:
This and the next paragraph both apply with either method of setting an selinux context, but it being placed here indicates it only applies to the first.

Maybe there should to be a new section below for these bits of info?